src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.trim().split(';');
```

Comment:
The variable name 'stylesArray' could be improved for better readability. Consider renaming it to reflect that it contains CSS declarations or rules.

Reply:
> The variable name 'stylesArray' could be improved for better readability. Consider renaming it to reflect that it contains CSS declarations or rules.

Good point